core/templates/pages/exploration-player-page/layout-directives/learner-view-info.directive.html

```diff
@@ -10,11 +13,35 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
 </ul>
 
 <style>
+  .oppia-navbar-breadcrumb {
+    padding-top: 10px;
+  }
+  .oppia-learner-navbar-container {
+    display: flex;
+  }
+  .lesson-info-section {
+    display: contents;
+  }
+  .oppia-navbar-title-container {
+    display: flex;
+    flex-direction: column;
+  }
   .oppia-exploration-h1 {
     color: #fff;
     display: inline;
-    font-size: 1em;
+    font-size: 18px;
     font-weight: normal;
+    margin-bottom: 1px;
+  }
+  .oppia-topic-title {
+    color: white;
+    display: flex;
+    font-size: 12px;
+    font-weight: 200;
+    padding-left: 26px;
+  }
+  .oppia-topic-title:hover {
+    color: white;
```

Comment:
Use the HEX value (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).